src/transformers/dynamic_module_utils.py

```diff
@@ -513,3 +514,45 @@ def _set_auto_map_in_config(_config):
         result.append(dest_file)
 
     return result
+
+
+def _raise_timeout_error(signum, frame):
+    raise ValueError(
+        "Loading this model requires you to execute the configuration file in that repo on your local machine. We "
+        "if it was okay but did not get an answer. Make sure you have read the code there to avoid malicious use, "
+        "then set the option `trust_remote_code=True` to remove this error."
+    )
+
+
+TIME_OUT_REMOTE_CODE = 30
```

Comment:
Is this something that you expect to be configurable, or not necessarily?

Reply:
Just for the CI